examples/preprocessing/plot_ica_from_raw.py

```diff
@@ -62,6 +62,7 @@
 
 show_picks = np.abs(scores).argsort()[::-1][:5]
 
+ica.plot_raw_components(raw, exclude=ecg_inds, title=title % 'ecg')
```

Comment:
This example is quite crazy. It opens 12 or 13 figures ! anything can be done about it? cc @dengemann